tests/test_batch.py

```diff
@@ -26,7 +26,7 @@ def test_batch_mode_00(self):
         Testing batch_mode: missing configuration file.
```

Comment:
there are no longer any tests for dutch_rivers in this case?

Reply:
Correct. Or phrased differently: the are no tests YET for the new Dutch_rivers.ini file. All existing tests still exist, but they are all based on the old Dutch_rivers_v1.ini configuration (which was previously called Dutch_rivers.ini). We'll start adding new cases using new functionality based on the new configuration file, but the new configuration file isn't final yet. The content will change significantly and almost all possible tests using it would have to get new references when it's updated to its final form.

Reply:
Note: I've now renamed the new Dutch_rivers.ini file to Dutch_rivers_v2.ini to make it more clear that this is a new file. The content is still subject to change. We can use a simpler file for basic unit testing.